first_boot-displaymanager-20160805.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "tags": [
+    "displaymanager"

```

Comment:
I just checked test_suites and job_groups on o3 and found no Tumbleweed scenario which is setting DOCRUN so the root password is never actually typed.

@richiejp I guess you set DOCRUN to make it work because otherwise user_import does not work because it does not expect the root password screen? If this is the case, please adapt the check in `products/opensuse/main.pm` to check for your new variable, too.

*and* we should think about why the test_suites `nice` and `doc` are not run anywhere.